public/pages/Dashboard/Container/DashboardOverview.tsx

```diff
@@ -203,75 +203,77 @@ export function DashboardOverview() {
   }, [selectedDetectorsName, selectedIndices, selectedDetectorStates]);
 
   return (
-    <Fragment>
-      <DashboardHeader hasDetectors={adState.totalDetectors > 0} />
-      {isLoadingDetectors ? (
-        <div>
-          <EuiLoadingSpinner size="s" />
-          &nbsp;&nbsp;
-          <EuiLoadingSpinner size="m" />
-          &nbsp;&nbsp;
-          <EuiLoadingSpinner size="l" />
-          &nbsp;&nbsp;
-          <EuiLoadingSpinner size="xl" />
-        </div>
-      ) : adState.totalDetectors === 0 ? (
-        <EmptyDashboard />
-      ) : (
-        <Fragment>
-          <EuiFlexGroup justifyContent="flexStart" gutterSize="s">
-            <EuiFlexItem>
-              <EuiComboBox
-                id="detectorFilter"
-                data-test-subj="detectorFilter"
-                placeholder={ALL_DETECTORS_MESSAGE}
-                options={getDetectorOptions(allDetectorList)}
-                onChange={handleDetectorsFilterChange}
-                selectedOptions={selectedDetectorsName.map(buildItemOption)}
-                isClearable={true}
-                fullWidth
-              />
-            </EuiFlexItem>
-            <EuiFlexItem>
-              <EuiComboBox
-                id="detectorStateFilter"
-                data-test-subj="detectorStateFilter"
-                placeholder={ALL_DETECTOR_STATES_MESSAGE}
-                options={getDetectorStateOptions()}
-                onChange={handleDetectorStateFilterChange}
-                selectedOptions={selectedDetectorStates.map(buildItemOption)}
-                isClearable={true}
-                fullWidth
-              />
-            </EuiFlexItem>
-            <EuiFlexItem>
-              <EuiComboBox
-                id="indicesFilter"
-                data-test-subj="indicesFilter"
-                placeholder={ALL_INDICES_MESSAGE}
-                options={getVisibleOptions(visibleIndices, visibleAliases)}
-                onChange={handleIndicesFilterChange}
-                selectedOptions={selectedIndices.map(buildItemOption)}
-                isClearable={true}
-                fullWidth
-              />
-            </EuiFlexItem>
-          </EuiFlexGroup>
-          <EuiSpacer />
-          <AnomaliesLiveChart selectedDetectors={currentDetectors} />
-          <EuiSpacer />
-          <EuiFlexGroup justifyContent="spaceBetween">
-            <EuiFlexItem grow={6}>
-              <AnomaliesDistributionChart
-                selectedDetectors={currentDetectors}
-              />
-            </EuiFlexItem>
-            <EuiFlexItem grow={3}>
-              <AnomalousDetectorsList selectedDetectors={currentDetectors} />
-            </EuiFlexItem>
-          </EuiFlexGroup>
-        </Fragment>
-      )}
-    </Fragment>
+    <div style={{ height: '1200px' }}>
```

Comment:
This is the only change for full screen issue. The root cause is that not all parent divs have specified 100% height or  explicit px for height. More details can be found here: https://stackoverflow.com/a/31728799

Reply:
I manually fix the height for dashboard to 1200 px, which looks appropriate on UI, without too much empty space at the bottom

**Without anomaly**
![Screen Shot 2020-09-04 at 11 33 22 AM](https://user-images.githubusercontent.com/59710443/92274644-7bf52f00-eea2-11ea-8c55-fb3790c17b83.png)

**With anomaly**
![Screen Shot 2020-09-04 at 11 34 48 AM](https://user-images.githubusercontent.com/59710443/92274766-aba43700-eea2-11ea-8f53-7e2d3ad6a7fc.png)